### PR TITLE
Account Unverified Email: Data Layer

### PR DESCRIPTION
**Pull request by jleandroperez**
Ref. #5086

#### Description:
In this PR we're adding new fields to the Data Model Mark 48:

- **AccountSettings**.**emailChangePending:** Boolean. Indicates if the email was changed, and there's a pending confirmation, or not.
- **AccountSettings**.**emailChangeAddress:** String. When available, indicates the new User's Email Address (with pending confirmation).

Suggestions for better field names would be more than welcome sir!

Needs review: @astralbodies 
Thanks in advance Aaron!

### CODE REVIEW

**Comment by jleandroperez** [comment]
@astralbodies conflict solved. Changes merged into Data Model 48, aaaand ready for review.

Thank you!

File: WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift [hunk 1]
```diff
@@ -9,6 +9,8 @@ extension ManagedAccountSettings {
 
     @NSManaged var username: String
     @NSManaged var email: String
+    @NSManaged var emailPendingAddress: String?
+    @NSManaged var emailPendingChange: Bool
```
Comment (astralbodies) [inline WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift]: I've not deviated from NSNumber before for boolean - seems safe?

**Comment by astralbodies** [comment]
Unit tests are failing because of a compilation error.

**Comment by astralbodies** [comment]
I fixed the broken unit test and tweaked MIGRATIONS.md to compile the changes into the one version. If the Bool is safe in Swift for the managed object, :shipit:

File: WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift (hunk 1 above)
Comment (jleandroperez) [inline WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift]: Confirmed, `Bool` and `Bool?` work great. Thank you Aaron!!

**Comment by jleandroperez** [comment]
@astralbodies thanks a lot for fixing that Test Case (++apologies for not catching that, shouldn't have happened).

**Thanks thanks thanks!!!**

**Pull request merged**